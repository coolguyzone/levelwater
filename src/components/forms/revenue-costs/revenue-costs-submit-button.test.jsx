import React from 'react';
import { shallow } from 'enzyme';
import { RevenueCostsSubmitButton } from './revenue-costs-submit-button.jsx';
import toJson from 'enzyme-to-json';

describe('revenue and costs submit button', () => {
  it('renders a snapshot', () => {
    const tree = shallow(<RevenueCostsSubmitButton />);
    expect(toJson(tree)).toMatchSnapshot();
  });
});
