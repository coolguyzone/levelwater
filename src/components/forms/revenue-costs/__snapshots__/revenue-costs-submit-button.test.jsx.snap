exports[`revenue and costs submit button renders a snapshot 1`] = `
<div
  className="column eight wide computer sixteen wide tablet"
  id="revenue-costs-submit-button">
  <button
    className="ui button">
    Sources >
  </button>
</div>
`;
